test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
Pasting from https://github.com/jquery/jquery/issues/3436#issuecomment-268031156:

Note that we have to be careful to not make the contents stop being inert once running `.contents()` on a template element wrapped in jQuery so we'll need a test that ensures this does not happen. For example, you could create a test with a `<template>` tag containing a script and `<img>` with an onload handler and make sure none of them fires.